hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapPartitionDestroyOperation.java

```diff
@@ -72,4 +79,24 @@ protected void readInternal(ObjectDataInput in) throws IOException {
         * It is local only operation and will never be serialized
         * */
     }
+
+    @Override
+    public boolean shouldBackup() {
+        return true;
+    }
+
+    @Override
+    public int getSyncBackupCount() {
+        return 0;
+    }
+
+    @Override
+    public int getAsyncBackupCount() {
+        return totalBackupCount;
```

Comment:
actually, you are right - it should be sync. as otherwise there might be operation reordering if there is a migration in progress.